test/local_sensitivity/sde_neural.jl

```diff
@@ -0,0 +1,102 @@
+using DiffEqFlux, Flux, LinearAlgebra
+using DiffEqNoiseProcess
+using StochasticDiffEq
+using Statistics
+using DiffEqSensitivity
+using DiffEqBase.EnsembleAnalysis
+
+function sys!(du, u, p, t)
+    r, e, μ, h, ph, z, i = p
+    du[1] = e * 0.5 * (5μ - u[1]) # nutrient input time series
+    du[2] = e * 0.05 * (10μ - u[2]) # grazer density time series
+    du[3] = 0.2 * exp(u[1]) - 0.05 * u[3] - r * u[3] / (h + u[3]) * u[4] # nutrient concentration
+    du[4] =
+        r * u[3] / (h + u[3]) * u[4] - 0.1 * u[4] -
+        0.02 * u[4]^z / (ph^z + u[4]^z) * exp(u[2] / 2.0) + i #Algae density
+end
+
+function noise!(du, u, p, t)
+    du[1] = p[end] # n
+    du[2] = p[end] # n
+    du[3] = 0.0
+    du[4] = 0.0
+end
+
+datasize = 10
+tspan = (0.0f0, 3.0f0)
+tsteps = range(tspan[1], tspan[2], length = datasize)
+u0 = Float32[1.0, 1.0, 1.0, 1.0]
+
+p_ = Float32[1.1, 1.0, 0.0, 2.0, 1.0, 1.0, 1e-6, 1.0]
+
+prob = SDEProblem(sys!, noise!, u0, tspan, p_)
+ensembleprob = EnsembleProblem(prob)
+
+solution = solve(
+    ensembleprob,
+    SOSRI(),
+    EnsembleThreads();  
+    trajectories = 1000,
```

Comment:
Probably best to drop the trajectories down to like 2 for the tests. The real values aren't important here.